test/yaml-rest-runner/src/test/java/org/elasticsearch/test/rest/yaml/section/ClientYamlTestSuiteTests.java

```diff
@@ -422,6 +429,49 @@ public void testParseSkipOs() throws Exception {
         assertThat(restTestSuite.getTestSections().get(0).getSkipSection().getFeatures(), containsInAnyOrder("skip_os"));
     }
 
+    public void testParseFileWithSingleTestSection() throws Exception {
```

Comment:
can we add a test that checks that this file is indeed logged?

Reply:
@fcofdez I just pushed a new integration test that checks that indeed the YAML test source code file is logged, along with the failing response, on a test failure. Please check again.